### PR TITLE
register page javascript, store and display form data in json

### PR DESCRIPTION
**Pull request by joeaoregan**
Updated the register page to use external JavaScript file for creating and managing the form

The submitted form data is stored and displayed using JSON

### CODE REVIEW

File: Project/register.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
+/*
+register.js
+J O'Regan
+15/11/2019
+*/
+
+var surveyArr = [];
```
Comment (joeaoregan) [inline Project/register.js]: Example comment: maybe add a comment explaining what the surveyArr array does

File: Project/register.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+	// Set the attributes of the JSON
+	surveyJSON.username = document.getElementById("usernameInput").value;
+	surveyJSON.password = document.getElementById("password1Input").value;
+	surveyJSON.firstname = document.getElementById("fnameInput").value;
+	surveyJSON.lastname = document.getElementById("lnameInput").value;
+	surveyJSON.email = document.getElementById("emailInput").value;
+	surveyJSON.amount = document.querySelector('input[type=radio]:checked').value;	
+	surveyJSON.favourite = document.getElementById("favouriteInput").value;
+	surveyJSON.newsletter = (document.getElementById("newsletterInput").checked) ? "Yes" : "No";
```
Comment (joeaoregan) [inline Project/register.js]: Example Comment: WTF are all these????

File: Project/register.js (hunk 1 above)
Comment (ianfitzpatrick52) [reply Project/register.js]: maybe add a comment explaining what the surveyArr array does?

File: Project/register.js (hunk 2 above)
Comment (ianfitzpatrick52) [reply Project/register.js]: What's going on with all of these?

File: Project/register.html [hunk 3]
```diff
@@ -12,215 +12,17 @@
...
+		<div id="menu" class="menu"></div><!-- menu.js: displays the menu here -->
 		<script src="menu.js"></script>
 			
 		<h1>Register</h1>	
-			
-		<div id="formdiv" class="register">
-			<form onsubmit="submitForm()">
-				<table>
-					<thead>
-						<th colspan="3">New User Registration</th>
```
Comment (ghost) [inline Project/register.html:26]: Might need you to explain whats going on with the <thead> <tr> <td> .

Comment (ghost) [reply Project/register.html:26]: thead/ th/ tr

Comment (joeaoregan) [reply Project/register.html:26]: table layout for the form https://www.w3schools.com/html/html_tables.asp

**Pull request merged**